packages/suite-desktop-api/src/messages.ts

```diff
@@ -2,7 +2,25 @@ import { ExtractUndefined } from './methods';
 
 export type SuiteThemeVariant = 'light' | 'dark' | 'system';
 
+export const TOR_STATUS_EVENT_TYPE = {
+    Enabled: 'Enabled',
+    Disabled: 'Disabled',
+    Disabling: 'Disabling',
+    Bootstrapping: 'Bootstrapping',
+    Misbehaving: 'Misbehaving',
```

Comment:
Better name?

Reply:
I still like the name since it means `to behave badly or improperly` and that is what I think we want to express there.

But if you give me other suggestion I will reconsider.